Package SDK (#313)
mozilla-ai/lumigator: lumigator/python/mzai/sdk/README.md

The SDK provides the communication and validation primitives needed to contact the Lumigator itself, either locally or remotely.

> You can install the lumigator SDK using the usual `pip` tool, taking care of selecting the appropriate environment. For example, when using `uv`:

Reviewer: ```suggestion
You can install the lumigator SDK  via `pip` directly or via `uv`:
```
